tachyon/math/finite_fields/prime_field_gpu_debug.h

```diff
@@ -394,7 +389,8 @@ class PrimeFieldGpuDebug final
 
   [[nodiscard]] constexpr static bool DoInverse(const PrimeFieldGpuDebug& a,
                                                 PrimeFieldGpuDebug& b) {
-    if (UNLIKELY(InvalidOperation(!a.IsZero(), "Inverse of zero attempted"))) {
+    if (UNLIKELY(!a.IsZero())) {
```

Comment:
```suggestion
    if (UNLIKELY(a.IsZero())) {
```
Ooops can you make a separate commit for this bug fix?